src/router/constants/studentRoutes.js

```diff
@@ -1,13 +1,13 @@
 export const studentRoutes = {
   navBar: {
-    findTutor: { label: 'findTutor', route: 'findTutor' },
+    findTutor: { label: 'findTutor', route: '/student/findTutor', nested: '/findTutor' },
```

Comment:
Why do you use "nested" if you have "route". And route you don't use anywhere.

Reply:
coz otherwise it would lead you to an error page. otherwise it doesn't work. i followed the pattern in the approutes.